src/system/irrigation.cpp

```diff
@@ -115,19 +115,19 @@ void IrrigationSystem::InitRelays()
                  .setTimeProvider(&timeProviders)
                  .setVoltageValvePin(IO_4)
                  .forPin(IO_0)
-                    .onDay(DAYS_PER_WEEK)
+                    .onDays(ALL_WEEK)
```

Comment:
how do you configure different days?
(SUNDAY | TUESDAY )  <-- like this?

Reply:
Yep